main.nf

```diff
@@ -928,36 +928,45 @@ workflow.onComplete {
                             .each { tmp ->
                                 if ( tmp.exists() == true ) {
                                     result = tmp.deleteDir();
-                                    log.info result ? "Deleted: $tmp" : "Can't delete: $tmp"
+                                    log.info result ? "Deleted: ${tmp}" : "Can't delete: ${tmp}"
                                     // Remove associated incomplete BLAST TSV file
                                     file("${wgd_dir}/*.blast.tsv", type: "file")
                                         .each { tsv ->
                                             if ( tsv.exists() == true ) {
                                                 result = tsv.delete()
-                                                log.info result ? "Deleted: $tsv" : "Can't delete: $tsv"
+                                                log.info result ? "Deleted: ${tsv}" : "Can't delete: ${tsv}"
                                             }
                                         }
                                 }
                             }
-                        // Remove Ks temporary folder, if any
+                        // Remove Ks temporary directory, if any
                         file("${wgd_dir}/*.ks_tmp", type: "dir")
                             .each { tmp ->
                                 if ( tmp.exists() == true ) {
                                     result = tmp.deleteDir();
-                                    log.info result ? "Deleted: $tmp" : "Can't delete: $tmp"
+                                    log.info result ? "Deleted: ${tmp}" : "Can't delete: ${tmp}"
                                 }
                             }
-                        // Remove i-ADHoRe temporary folder, if any (applicable only to paralog_distributions)
+                        // Remove i-ADHoRe temporary directory, if any (applicable only to paralog_distributions)
                         file("${wgd_dir}/*.ks_anchors_tmp", type: "dir")
                             .each { tmp ->
                                 if ( tmp.exists() == true ) {
                                     result = tmp.deleteDir();
-                                    log.info result ? "Deleted: $tmp" : "Can't delete: $tmp"
+                                    log.info result ? "Deleted: ${tmp}" : "Can't delete: ${tmp}"
                                 }
                             }
                     }
             }
 
+            // Remove "core" files generated when the submitted job doesn't have enough memory
+            file("${workflow.launchDir}/core.*")
```

Comment:
To be more specific I would here also select on only numbers after the period, not just any character(s).

Reply:
Is it always going to be numbers after the dot?

Reply:
Not necessarily, but likely (unless the naming of core files has been changed by admins, but could then also not include "core"). If you don't want to delete any other files named `core.*` by mistake I would add the selection on numbers. If you think it's more unlikely to have other files named `core.*` then you can leave it like it is.

Reply:
Done!